html/news/Cinema_Perpetuum_Mobile_2014.htm

```diff
@@ -948,145 +947,116 @@ <h3 class="newsblock_headerlink">
+<footer id="contact" class="footer container-fluid">
+    <div class="container dark">
+        <div class="row footer-menu">
+            <div class="span4 pagination-centered">
+                international<br>
+                short film festival<br>
+                <strong>Cinema Perpetuum Mobile</strong><br>
+                Spring 2014<br>
+                Minsk, Belarus
             </div>
-        </div>
-    </div>
-
-
-    <footer id="contact" class="footer container-fluid">
-        <div class="container dark">
-            <div class="row footer-menu">
-                <div class="span4 pagination-centered">
-                    international<br>
-                    short film festival<br>
-                    <strong>Cinema Perpetuum Mobile</strong><br>
-                    Spring 2014<br>
-                    Minsk, Belarus
-                </div>
-                <div class="span8">
-                    <ul>
-                        <li>
-                            <a href="http://filmfest.by/en/">Home</a>
-                        </li>
-                        <li>
-                            Festival:
-                            <ul class="footer-links">
-                                <li><a href="http://filmfest.by/en/2012/">2012: good memories</a></li>
-                                <li><a href="http://filmfest.by/en/2013/">2013: good memories</a></li>
-                                <li><a href="http://filmfest.by/en/rules/">Regulations</a></li>
-                                <li><a href="http://filmfest.by/en/submit/">Submit your film!</a></li>
-                            </ul>
-                        </li>
-                        <li>
-                            Volunteers:
-                            <ul class="footer-links">
-                                <li><a href="http://filmfest.by/en/vklyuchajsya/">Join in</a></li>
-                                <li><a href="http://groups.google.com/group/cpm2015">Discussion group</a></li>
-                                <li><a href="http://filmfest.by/ru/2013/page/volunteer_form">Volunteer form</a>
-                                </li>
-                            </ul>
-                        </li>
-                        <li>
-                            <a href="http://filmfest.by/en/cpm/partners/">Partners</a>
-                        </li>
-                        <li>
-                            Press-center:
-                            <ul class="footer-links">
-                                <li><a href="http://filmfest.by/en/press-kit/">Press-kit</a></li>
-                                <li><a href="http://filmfest.by/en/media/">Festival in media</a></li>
-                            </ul>
-                        </li>
-                        <li>
-                            <a href="http://filmfest.by/contact/">Contacts</a>
-                        </li>
-                    </ul>
-                </div>
+            <div class="span8">
```

Comment:
I don't format this part of markup. In markup on laptop this part is thue. But i don't inderstand - why footer is not defined.